hypervisor/arch/x86/cpu_state_tbl.c

```diff
@@ -34,6 +34,23 @@ struct cpu_cx_data cx_a3960[] = {
 	{{SPACE_SYSTEM_IO, 0x8, 0, 0, 0x419}, 0x3, 0x96, 0x0A}  /* C3 */
 };
 
+/* The table includes cpu px info of Intel A3950 SoC */
+struct cpu_px_data px_a3950[] = {
```

Comment:
static?

Reply:
OK.
Can I keep it in this patch and submit another patch to change all data struct to "static const"?
We have other data defined without "static" in the file...